core/store/orm/schema.go

```diff
@@ -22,7 +22,8 @@ type ConfigSchema struct {
 	AuthenticatedRateLimit                     int64                         `env:"AUTHENTICATED_RATE_LIMIT" default:"1000"`
 	AuthenticatedRateLimitPeriod               time.Duration                 `env:"AUTHENTICATED_RATE_LIMIT_PERIOD" default:"1m"`
 	BalanceMonitorEnabled                      bool                          `env:"BALANCE_MONITOR_ENABLED" default:"true"`
-	BlockBackfillDepth                         string                        `env:"BLOCK_BACKFILL_DEPTH" default:"10"`
+	BlockBackfillDepth                         uint64                        `env:"BLOCK_BACKFILL_DEPTH" default:"10"`
+	BlockBackfillMode                          string                        `env:"BLOCK_BACKFILL_MODE"`
```

Comment:
Maybe simpler/cleaner to just call this SkipBackfill and make it a bool (default to false)?

Reply:
I was thinking it will be more flexible in the future, like with GasEstimatorMode recently introduced. But yeah probably better to make it more narrow until we have more idea about the future here.

Reply:
so i named it now 'BlockBackfillSkip'.